release_dashboard/lib/widgets/conductor_status.dart

```diff
@@ -236,6 +237,10 @@ class RepoInfoExpansionState extends State<RepoInfoExpansion> {
 
   @override
   Widget build(BuildContext context) {
+    Map<String, int> URLelements = <String, int>{
+      'engine luci dashboard': 4,
+      'framework luci dashboard': 4,
```

Comment:
I wasn't sure how to assign the enums as you have indicated previously, I tried:
```
  static Enum URLelements2 {
    'engine luci dashboard'=4,
    'framework luci dashboard'=4,
  }

```
I got this error on `URLelements2`
```
Methods must have an explicit list of parameters.
Try adding a parameter list.dart(missing_method_parameters)
The body might complete normally, causing 'null' to be returned, but the return type is a potentially non-nullable type.
Try adding either a return or a throw statement at the end.dartbody_might_complete_normally
Enum URLelements2()
package:conductor_ui/widgets/conductor_status.dart
```
and on `'engine luci dashboard'` and `'framework luci dashboard'`
```
Illegal assignment to non-assignable expression.dart(illegal_assignment_to_non_assignable)
Missing selector such as '.identifier' or '[0]'.
Try adding a selector.dart(missing_assignable_selector)
```

Reply:
This should work:

```
enum ClickableElements {
  luciDashboard = 4;
}
```

Thinking about it some more, since this is for the repo info, have you thought about just checking if the value starts with `http`, and if so making it clickable?